ISBN/src/d/ISBN.php

```diff
@@ -110,6 +119,13 @@ private function splitIsbn(string $isbn) : array
         return preg_split("/[-\040]+/", $isbn);
     }
 
+    private function ensureRightLength(string $cleanIsbn)
```

Comment:
gut! was passiert eigentlich, wenn eine ISBN nicht nur aus zahlen besteht? in der Methode ensureRightGroupNumber() wird ja lustig drauf los gerechnet...